src/cli/tws/approve.js

```diff
@@ -0,0 +1,9 @@
+const logger = require('../../logger')
+
+const { approveCommand } = require('../../commands/commandList')
+
+module.exports = {
```

Comment:
Hey @ceres3idoo could be convenient to place this kind of command in a command list related to the ERC20 token functionalities. So we can do the same for the check of the allowance.

Reply:
We are going to do this in a later refactor.